File: GTTS/INL.TrainingService.Models/DeleteTrainingEventParticipant_Param.cs
using INL.Services.Models;

namespace INL.TrainingService.Models
{
    public class DeleteTrainingEventParticipant_Param : BaseParam, IDeleteTrainingEventParticipant_Param
    {
        public DeleteTrainingEventParticipant_Param() : base() { }

        public long TrainingEventID { get; set; }
        public long ParticipantID { get; set; }
        public string ParticipantType { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.TrainingService.Models/delete-training-event-participant_param.ts


import { IDeleteTrainingEventParticipant_Param } from './idelete-training-event-participant_param';

export class DeleteTrainingEventParticipant_Param implements IDeleteTrainingEventParticipant_Param {
  
	public TrainingEventID: number = 0;
	public ParticipantID: number = 0;
	public ParticipantType: string = "";
  
	public ErrorMessages: string[] = null;
}






